ckanext/ontario_theme/plugin.py

```diff
@@ -78,176 +80,101 @@ def csv_dump():
-                "URL": {
-                    "pattern": "^url$"
-                },
-                "Technical Title EN": {
-                    "pattern": ["^technical_title$", "^en$"]
-                },
-                "Technical Title FR": {
-                    "pattern": ["^technical_title$", "^fr$"]
-                },
-                "Node Id": {
-                    "pattern": "^node_id$"
-                },
-                "Removal Rationale EN": {
-                    "pattern": ["^removal_rationale$", "^en$"]
-                },
-                "Removal Rationale FR": {
-                    "pattern": ["^removal_rationale$", "^fr$"]
-                },
-                "Update Frequency": {
-                    "pattern": "^update_frequency$"
-                }
-              }
+    columns = OrderedDict([
+        ("Id", {
+            "pattern": "^id$"
+        }),
+        ("Name", {
+            "pattern": "^name$"
+        }),
+        ("Title EN", {
+            "pattern": ["^title_translated$", "^en$"]
+        }),
+        ("Notes EN", {
+            "pattern": ["^notes_translated$", "^en$"]
+        }),
+        ("Organization Title", {
+            "pattern": ["^organization$", "^title$"]
+        }),
+        ("Access Level", {
+            "pattern": "^access_level$"
+        }),
+        ("Type", {
+            "pattern": "^type$"
+        }),
+        ("Update Frequency", {
+            "pattern": "^update_frequency$"
+        }),
+        ("Metadata Created", {
+            "pattern": "^metadata_created$"
+        }),
+        ("Metadata Modified", {
+            "pattern": "^metadata_modified$"
+        }),
+        ("License Title", {
+            "pattern": "^license_title$"
+        }),
+        ("Keywords EN", {
+            "pattern": ["^keywords$", "^en$"]
+        }),
+        ("Package Date Opened", {
+            "pattern": "^opened_date$"
+        }),
+        ("Package Last Validated Date", {
+            "pattern": "^current_as_of$"
+        }),
+        ("Exemption", {
+            "pattern": "^exemption$"
+        }),
+        ("Exemption Rationale EN", {
+            "pattern": ["^exemption_rationale$", "^en$"]
+        }),
+        ("Geographic Coverage EN", {
+            "pattern": ["^geographic_coverage_translated$", "^en$"]
+        }),
+        ("Resources Format", {
+            "pattern": ["^resources$", "^format$"],
+            "deduplicate": "true"
+        }),
+        ("Num Resources", {
+            "pattern": "^num_resources$"
+        }),
+        ("Title FR", {
+            "pattern": ["^title_translated$", "^fr$"]
+        }),
+        ("Geographic Coverage FR", {
+            "pattern": ["^geographic_coverage_translated$", "^fr$"]
+        }),
+        ("Exemption Rationale FR", {
+            "pattern": ["^exemption_rationale$", "^fr$"]
+        }),
+        ("Keywords FR", {
+            "pattern": ["^keywords$", "^fr$"]
+        }),
+        ("Contains geopgrapic markers", {
```

Comment:
Can we ditch "contains geographic markers", "data range start", "data range end", and "licence id"?